src/components/editor/elements/uploads/element-upload-canvas.tsx

```
@@ -0,0 +1,37 @@
+        rotation={element.rotation}
+        width={element.width}
+        height={element.height}
+        draggable
+        onDragEnd={(e) => onDragEnd(element.id, e.target)}
+        onMouseDown={(e) => onSelect(element.id, e.target)}
+        onClick={(e) => onSelect(element.id, e.target)}
+        onTap={(e) => onSelect(element.id, e.target)}
+      />
+    );
+  }
+);
```

ESLint

[error] Component definition is missing display name

(react/display-name)